README.rst

```diff
@@ -0,0 +1,151 @@
+Nameko eventlog dispatcher
+==========================
+
+.. pull-quote::
+
+    `Nameko <http://nameko.readthedocs.org>`_ dependency provider that
+    dispatches log data using ``Events`` (Pub-Sub).
+
+
+.. image:: https://travis-ci.org/sohonetlabs/nameko-eventlog-dispatcher.svg?branch=master
+
+
+Usage
+-----
+
+Dispatching event log data
+``````````````````````````
+
+Include the ``EventLogDispatcher`` dependency in your service class:
+
+.. code-block:: python
+
+    from nameko.rpc import rpc
+    from nameko_eventlog_dispatcher import EventLogDispatcher
+
+
+    class FooService:
+
+        name = 'foo'
+
+        eventlog_dispatcher = EventLogDispatcher()
+
+        @rpc
+        def foo_method(self):
+            self.eventlog_dispatcher('foo_event_type', {'value': 1})
+
+``event_type`` and  some ``event_data`` (optional) will be provided as
+arguments. ``event_data`` must contain serializable data.
```

Comment:
serializable is too generic, we should specify JSON serializable, YAML serializable, something like that.